front-end/README.md

```diff
@@ -6,6 +6,10 @@ This project was bootstrapped with [Create React App](https://github.com/faceboo
 
 In the project directory, you can run:
 
+### `npm install`
+
+Installs npm, allowing the following commands.
+
 ### `npm start`
 
 Runs the app in the development mode.\
```

Comment:
Oh, I also noticed this stray backslash that I think doesn't do anything, could you remove this too?

Reply:
Yikes, I still don't have a handle on GitHub--the changes requested are more for the comment above than this.

Reply:
I believe that backslash actually functions as a linebreak.